versions.lock

```diff
@@ -33,7 +34,6 @@ ch.qos.logback:logback-core:1.1.2 (1 constraints: 050dfb1c)
 com.squareup.okhttp3:okhttp:4.0.0 (1 constraints: 0605ff35)
 com.squareup.okio:okio:2.2.2 (1 constraints: 500c29fd)
 io.undertow:undertow-core:2.0.20.Final (1 constraints: 4e07ff60)
-net.bytebuddy:byte-buddy:1.9.10 (1 constraints: 450b52de)
 net.bytebuddy:byte-buddy-agent:1.9.10 (1 constraints: 450b52de)
```

Comment:
I don't think this will cause issues, but curious if we should bump this ([possibly upstream](https://github.com/mockito/mockito/blame/25119e12946d5e79a003b4e778436a647a2faef4/gradle/dependencies.gradle#L7))?

```
$ gw why --hash 450b52de
net.bytebuddy:byte-buddy-agent:1.9.10
        org.mockito:mockito-core -> 1.9.10
```

Reply:
https://github.com/mockito/mockito/pull/1772